contracts/bridge/BridgeOperator.sol

```diff
@@ -32,12 +34,13 @@ contract BridgeOperator is Ownable {
     }
 
     // voteCommon handles common functionality for voting.
-    function voteCommon(VoteType voteType, bytes32 _voteKey, uint64 _requestNonce)
+    function voteCommon(VoteType voteType, bytes32 _voteKey)
         internal
         returns(bool)
     {
         if (!votes[_voteKey][msg.sender]) {
             votes[_voteKey][msg.sender] = true;
+            require(votesCounts[_voteKey] < votesCounts[_voteKey] + 1);
```

Comment:
@KimKyungup What does it for? Overflow checking?

Reply:
Right. it's overflow checking.